app/lib/actions/candlepin/reindex_pool_subscription_handler.rb

```diff
@@ -30,13 +30,13 @@ def handle(message)
         wrapped_message = MessageWrapper.new(message)
         case message.subject
         when /entitlement\.created/
-          reindex_pool(wrapped_message.content['referenceId'])
+          import_pool(wrapped_message.content['referenceId'])
```

Comment:
This file and class look like they could use a name overhaul since we aren't "reindexing"